### PR TITLE
Cobbweb/more readme fixes

### PR DESCRIPTION
**Pull request by cobbweb**
Can be [previewed here](https://github.com/cobbweb/Vim/tree/cobbweb/more-readme-fixes).

Added a silly badge just for you @johnfn (let me know when/if to remove this)

### CODE REVIEW

**Comment by jpoon** [comment]
I may have missed the joke. What's this potato badge?

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-<h1 align="center"><img src="./images/icon.png" height="128"><br>VSCodeVim</h1>
+<h1 align="center"><img src="https://raw.githubusercontent.com/VSCodeVim/Vim/master/images/icon.png" height="128"><br>VSCodeVim</h1>
 <p align="center"><strong>Vim emulation for Visual Studio Code.</strong></p>
-<p align="center"><a href="http://aka.ms/vscodevim"><img src="http://vsmarketplacebadge.apphb.com/version/vscodevim.vim.svg" alt="Version"></a><a href="https://travis-ci.org/VSCodeVim/Vim"><img src="https://travis-ci.org/VSCodeVim/Vim.svg?branch=master" alt="Build Status"></a><a href="https://vscodevim-slackin.azurewebsites.net"><img src="https://vscodevim-slackin.azurewebsites.net/badge.svg" alt="Slack Status"></a></p>
+<p align="center"><a href="http://aka.ms/vscodevim"><img src="http://vsmarketplacebadge.apphb.com/version/vscodevim.vim.svg" alt="Version"></a> <a href="https://github.com/VSCodeVim/Vim/pull/1643"><img src="https://img.shields.io/badge/potato-procrastinating-208ecd.svg" alt="Build Status"></a> <a href="https://vscodevim-slackin.azurewebsites.net"><img src="https://vscodevim-slackin.azurewebsites.net/badge.svg" alt="Slack Status"></a></p>
```
Comment (jpoon) [inline README.md]: > https://github.com/VSCodeVim/Vim/pull/1643

This should point to the main repo and not to the PR.

**Comment by xconverge** [comment]
Haha yea we can remove the potato badge

**Comment by johnfn** [comment]
Lol. I was talking with cobbwebb and we were talking about how every repo has a bunch of badges that don't really mean anything. We thought it would be funny to, on the end of our badge list, have a badge that was totally pointless. 

It would also be a great indication of VSCodeVim "culture". 😁

**Comment by cobbweb** [comment]
Sorry, off grid traveling at the moment... Won't get to fix this up for a week or so. Just want to confirm the build status badge should still be removed?

**Comment by Chillee** [comment]
@cobbweb Nah I think we should keep the build status page. @jpoon put in some work to fix all the failing tests, so a failing travis build should actually mean something now.

**Comment by cobbweb** [comment]
Sorry for delay. Branch fixed up: left build badge as-is, fixed logo src, fixed another heading bug

**Pull request merged**